recipe/bld.bat

```diff
@@ -0,0 +1,7 @@
+%PYTHON% build/build.py
+bazel clean --expunge
+bazel shutdown
+
+%PYTHON% -m pip install --find-links=dist jaxlib --no-build-isolation --no-deps
+
+copy %PREFIX%\Lib\site-packages\jaxlib\mlir\_mlir_libs\*.dll %LIBRARY_BIN%\
```

Comment:
we've got mlir on win, would adding that help?

Reply:
I don't think it will help until I can un-vendor mlir (it takes a minimum of 5 hours to try it) . I plan on taking a pass on unvendoring on the next version.

Reply:
oh, right. yep